adapters/repos/db/vector/hnsw/search.go

```diff
@@ -177,26 +185,37 @@ func (h *hnsw) cacheSize() int64 {
 	return size
 }
 
-func (h *hnsw) acornParams(allowList helpers.AllowList) (bool, int) {
-	useAcorn := h.acornSearch.Load()
-	var M int
+func (h *hnsw) acornParams(allowList helpers.AllowList) bool {
```

Comment:
Since we don't return `M` now can we rename `acornParams` be renamed `acornEnabled` or something?